ddtrace/internal/compat.py

```diff
@@ -287,7 +287,7 @@ def ensure_pep562(module_name):
 try:
     from collections.abc import Iterable  # noqa
 except ImportError:
-    from collections import Iterable  # type: ignore[no-redef, attr-defined]  # noqa
+    from collections import Iterable  # type: ignore[attr-defined]  # noqa
```

Comment:
For some reason this `no-redef` was causing linting failures on this branch. That's why this otherwise-unrelated change is present here.